crates/air_r_formatter/src/r/auxiliary/braced_expressions.rs

```diff
@@ -20,24 +20,61 @@ impl FormatNodeRule<RBracedExpressions> for FormatRBracedExpressions {
             r_curly_token,
         } = node.as_fields();
 
-        // Check if we are formatting empty braces, like `{ }`
-        if expressions.is_empty() {
-            return fmt_empty(&l_curly_token, &r_curly_token, node, f);
+        match braced_expressions_variant(node, f.comments()) {
```

Comment:
Implementation wise the most important thing to look at is this nice categorization helper, `braced_expressions_variant()`.

We use it here and over in `call_arguments.rs`. Splitting `RBracedExpressions` into these 4 categories is _just_ the right amount of categorization to be maximally reusable